### PR TITLE
feat(Root): add Switch demos

### LINKED ISSUE #1242: Switch Demo #SUIC-307
# Acceptance Criteria
The Switcher demo viewed on sui-studio is not following the agreements.
It can be previewed on the following link:
[sui-studio switch DEMO](https://sui-components.now.sh/workbench/atom/switch/demo)

# Definition Of DONE
- [ ] Add description
- [ ] Add a category “states” and move Disabled there

> ### Jira Task
> [SUIC-307](https://jira.scmspain.com/browse/SUIC-307)

### PR DESCRIPTION
**Pull request by andresin87**
ISSUES CLOSED: #1242

## atom/switch
**TASK**: [#1242](https://github.com/SUI-Components/sui-components/issues/1242) 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Add demo

### Description, Motivation and Context

- [x] Add description
- [x] Add a category “states” and move Disabled there

### CODE REVIEW

File: demo/atom/switch/playground [hunk 1]
```diff
@@ -27,6 +27,12 @@ const ValueParameter = () => {
 return (
 <div>
   <h1>Switch</h1>
+  <p class="sui-Studio-h4">
+    The switch is the radio button when there’re only 2 exclusive options. “On/off” is a common and clear example for explaining this component.
+    In order to collect the result of this switch there is a callback `onToggle`, this callback receives a flag on `true` if select is active. If you're using a `select` type of this component, `false` means the first option and `true` the second one.
+    There are several two sizes for this component: `default` and `large`.
```
Comment (desko27) [inline demo/atom/switch/playground]:
```suggestion
    There are two sizes for this component: `default` and `large`.
```

**Review by desko27 — APPROVED** [review]

File: demo/atom/switch/playground [hunk 2]
```diff
@@ -27,6 +27,12 @@ const ValueParameter = () => {
 return (
 <div>
   <h1>Switch</h1>
+  <p class="sui-Studio-h4">
```
Comment (turolopezsanabria) [inline demo/atom/switch/playground:30]: I think it has to be `className`

File: demo/atom/switch/playground (hunk 1 above)
Comment (turolopezsanabria) [inline demo/atom/switch/playground]: remove "several", just 2

Comment (andresin87) [reply demo/atom/switch/playground]: done

File: demo/atom/switch/playground [hunk 3]
```diff
@@ -27,6 +27,12 @@ const ValueParameter = () => {
...
+    The switch is the radio button when there’re only 2 exclusive options. “On/off” is a common and clear example for explaining this component.
+    In order to collect the result of this switch there is a callback `onToggle`, this callback receives a flag on `true` if select is active. If you're using a `select` type of this component, `false` means the first option and `true` the second one.
+    There are two sizes for this component: `default` and `large`.
+  </p>
+
   <h3>Styles</h3>
```
Comment (turolopezsanabria) [inline demo/atom/switch/playground]:
```suggestion
  <h3>Sizes</h3>
```

File: demo/atom/switch/playground [hunk 4]
```diff
@@ -43,6 +49,15 @@ return (
     </div>
```
Comment (turolopezsanabria) [inline demo/atom/switch/playground]: I suggest removing "Disabled" from this section, as it's already present in the next section

**Review by midudev — APPROVED** [review]

**Review by turolopezsanabria — APPROVED** [review]

**Pull request merged**